.circleci/config.yml

```diff
@@ -99,15 +99,45 @@ jobs:
       - set-up-packages
       - run: npm run env start && npm run test:e2e
 
+  svn-deploy:
+    executor:
+      name: php
+    working_directory: ~/project/svn_deploy
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - checkout
+      - run:
+          command: |
+            BUILD_VERSION=$(grep 'Version:' genesis-custom-blocks.php | cut -f4 -d' ')
+            sudo apt-get update && sudo apt-get install subversion
+            svn co https://plugins.svn.wordpress.org/genesis-custom-blocks package
```

Comment:
One thing [we do in AB](https://github.com/studiopress/atomic-blocks/blob/0d1808b57bcb4caf99ce6de419be1876fac7d02f/.circleci/config.yml#L47) is use the `--depth=empty` option when doing a checkout. The reason was the amount of time it takes to do a checkout once you have a sufficient number of releases out there. The svn server is just so slow sometimes. Another option is to prune tags every now and then, which [wp.org recommends](https://developer.wordpress.org/plugins/wordpress-org/plugin-developer-faq/#how-many-old-releases-should-i-keep-in-svn) - we just never got around to implementing that.

Not a blocker, but just mentioning for future consideration.

Reply:
Good idea, I'll make a commit for that

Reply:
Hm, maybe pruning would be good when there are several tags.

When adding `--depth=empty`, `circleci local execute --job svn-deploy` shows this message:

```
Searching tree conflict details for 'assets' in repository:
Checking r2372653... done
Tree conflict on 'assets':
A new directory appeared during update to r2378985; it was added by plugin-master in r2372653.
A directory scheduled to be added to the repository in the next commit was found in the working copy.
Select: (p) Postpone, (r) Mark as resolved, (i) Ignore incoming addition,
        (m) Merge the directories, (h) Help, (q) Quit resolution: 
```

There's probably a way around that, but it might be complex.

Reply:
@kienstra yeah I think for now let's leave it as is. We'll have to update some other parts of the script to get this in.